server/common/api/models.py

```diff
@@ -12,6 +12,7 @@ class Class(models.Model):
 	minGuestCount = models.IntegerField(default = 4)
 	maxGuestCount = models.IntegerField(default = 8)
 	price = models.FloatField()
+	classRating = models.FloatField(default = 0.0)
```

Comment:
set the boundary of the class rating
https://docs.djangoproject.com/en/dev/ref/validators/#minvaluevalidator

Reply:
thanks to your advice!!
I set boundary from 0.0 to 5.0